src/main/java/book/store/onlinebookstore/config/SecurityConfig.java

```diff
@@ -36,7 +36,10 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(
                         auth -> auth
-                                .requestMatchers("/api/auth/**")
+                                .requestMatchers("/api/auth/**",
+                                        "/error",

```

Comment:
what is this path for?

Reply:
without it server responses to a USER_ROLE who wants to create/update/delete with 401 status, with it with 403